DataLayer/Wearable/src/main/java/com/example/android/wearable/datalayer/ConnectedNodesScreen.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.wearable.datalayer
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.google.android.horologist.annotations.ExperimentalHorologistApi
+import com.google.android.horologist.compose.layout.ScalingLazyColumnDefaults
+import com.google.android.horologist.compose.layout.ScalingLazyColumnDefaults.ItemType
+import com.google.android.horologist.compose.layout.rememberResponsiveColumnState
+
+@OptIn(ExperimentalHorologistApi::class)
+@Composable
+fun ConnectedNodesScreen(
+    viewModel: NodesViewModel
+) {
+    val state by viewModel.state.collectAsStateWithLifecycle()
+
+    val columnState = rememberResponsiveColumnState(
```

Comment:
I'd suggest moving this down into NodesScreen. It knows about the content structure, and doesn't need a param.